caas/kubernetes/provider/admissionregistration_test.go

```diff
@@ -43,7 +44,7 @@ func (s *K8sBrokerSuite) assertMutatingWebhookConfigurations(c *gc.C, cfgs []k8s
 	statefulSetArg := &appsv1.StatefulSet{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:   "app-name",
-			Labels: map[string]string{"juju-app": "app-name"},
+			Labels: utils.LabelsForApp("app-name", false),
```

Comment:
For test assertion, it's better to use map[string]string{} rather than the computed result

Reply:
done